Dockerfile

```diff
@@ -50,6 +50,9 @@ CMD ["react-scripts", "start"]
 FROM appbase as staticbuilder
 # ===================================
 
+ARG REACT_APP_API_URI
+ARG REACT_APP_OIDC_AUTHORITY
+
```

Comment:
are you sure you copied everything from ui dockerfile?

Reply:
not anymore

Reply:
now they should be the same